sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -273,6 +273,14 @@ def test_importances():
 def test_error():
     # Test that it gives proper exception on deficient input.
 
+    reg = AdaBoostRegressor(loss="foo")
+    with pytest.raises(ValueError):
+        reg.fit(X, y_class)
+
+    clf = AdaBoostClassifier(algorithm="foo")
+    with pytest.raises(ValueError):
+        clf.fit(X, y_class)
```

Comment:
@reshamas: I should have been clearer.

Here for instance, there's a check for `ValueErrors` being raised but their error messages aren't checked.

Reply:
Should we re-open this one?  Error messages are important and helpful, correct?

Reply:
Here we go: https://github.com/scikit-learn/scikit-learn/pull/22144